R/combine-arrays.R

```diff
@@ -0,0 +1,52 @@
+##' Combine array variables into a new array
+##'
+##' This function lets you combine array variables into a new array
+##' Arguments include at least two array variables as well as any metadata
+##' for the new variable that is being created. This function uses the data from 
+##' the arrays in the order that they are given. If the first variable is NA for a given 
+##' row of a subvariable, it will use the data from the second variable and so on.
+##' @param variable 1 to combine
+##' @param variable 2 to combine
+##' @param other desired attributes such as new variable alias, name, 
+##' description as well as any other variables you would like to combine
+##' @return a new variable with combine values from the arrays
+##' @export
+
```

Comment:
I think you need to eliminate this space and run `make doc` again. I suspect this is why you don't have an entry in your NAMESPACE file for these functions.